docs/_data/2019.australia.speakers.yaml

```diff
@@ -117,63 +117,32 @@
     twitter: ''
     website: ''
   title: Rethinking Release Notes
-- abstract: '<p>We have been asked to do this very much lately. Customers don''t have
-    time or patience to sit down and read our User''s Guides, they would rather watch
-    a short "How to" video. </p>
+- abstract: '<p>Minimum Viable Product or MVP is a development technique where a
```

Comment:
Is this right? Looks like it might be muddled.

Reply:
I just removed the video talk as the speaker cannot make it.

Reply:
@plaindocs - good for a merge now?